.tekton/create-snapshot-task.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-snapshot
+  namespace: rh-acs-tenant
+spec:
+  description: Creates a snapshot based on the component names, image refs and Git information passed to the task.
+  params:
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the application source code.
+    type: string
```

Comment:
I feel like I may forget it in the future so I'll add a comment now. Once you'll inline `render_snapshot.py`, it seems it won't be necessary to use the git repo at all. Let's not forget to delete this `SOURCE_ARTIFACT` and the corresponding `use-trusted-artifact` step.